ignite-base/App/Themes/Colors.js

```diff
@@ -22,4 +24,8 @@ const colors = {
   drawer: 'rgba(30, 30, 29, 0.95)'
 }
 
+const colors = merge(igniteColors, {
+  // add colors for your Ignited app here
+})
+
```

Comment:
I'm playing with the idea in my head of all the ignite colors (and other flavors, like metrics, strings etc) coming from a node module, so that they would not be a distracting 20line block in this code?  Or is that too petty?